extensions/base/suite.go

```diff
@@ -18,17 +18,25 @@ package base
 
 import (
 	"github.com/networkservicemesh/integration-tests/extensions/checkout"
+	"github.com/networkservicemesh/integration-tests/extensions/logs"
+	"github.com/networkservicemesh/integration-tests/extensions/multisuite"
 )
 
 // Suite is a base suite for generating tests. Contains extensions that can be used for assertion and automation goals.
 type Suite struct {
-	checkout.Suite
-	// Add other extensions here
+	multisuite.Suite
 }
 
 func (s *Suite) SetupSuite() {
-	s.Repository = "networkservicemesh/deployments-k8s"
-	s.Version = "25830bce"
-	s.Dir = "../" // Note: this should be synced with input parameters in gen.go file
+	// Add other extensions here
+	s.Suite.WithSuits(
+		checkout.NewSuite(
+			"networkservicemesh/deployments-k8s",
+			"25830bce",
+			"../", // Note: this should be synced with input parameters in gen.go file
+		),
+		new(logs.Suite))
+
```

Comment:
Please let's do not use these constructions. It reduces code readability. 
Now `checkout.NewSuite` wants a few strange strings and it can confuse a bit.
Moreover, it will break automation from NSMBot.
I would recommend using previous behavior.

```suggestion
  var suite1 ...
  suite1.Property = ...
  s.Suite = multisuite.Of(suite1, suite2) ...
```

Reply:
Done